packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -182,10 +165,9 @@ export class DocumentService
 		assert(!!this.documentStorageService, 0x0b1 /* "Storage service not initialized" */);
 
 		const getRestWrapper = async (): Promise<RestWrapper> => {
-			const shouldUpdateDiscoveredSessionInfo = this.shouldUpdateDiscoveredSessionInfo();
+			const refreshed = await this.maybeRefreshDiscovery();
```

Comment:
>refreshed [](http://example.com/codeflow?start=9&length=9)

isRefreshed`

Reply:
`isRefreshed` sounds a bit wonky to me. I'll stick with `refreshed`

Reply:
Sure. The idea was to help variable name depict its data-type. Commonly followed practices across OPG codebase. It's alright to keep it as is, as I don't see FF following strict coding conventions anywhere.